Source/WebCore/css/SelectorChecker.cpp

```diff
@@ -1503,6 +1525,44 @@ bool SelectorChecker::checkScrollbarPseudoClass(const CheckingContext& checkingC
     }
 }
 
+bool SelectorChecker::checkViewTransitionPseudoClass(const CheckingContext& checkingContext, const Element& element, const CSSSelector& selector) const
+{
+    ASSERT(selector.match() == CSSSelector::Match::PseudoClass);
+
+    if (selector.pseudoClass() != CSSSelector::PseudoClass::OnlyChild)
+        return false;
+
+    switch (checkingContext.pseudoId) {
+    case PseudoId::ViewTransition:
+        return false;
+    case PseudoId::ViewTransitionGroup: {
+        if (RefPtr activeViewTransition = element.document().activeViewTransition()) {
+            auto* capturedElement = activeViewTransition->namedElements().find(checkingContext.pseudoElementNameArgument);
```

Comment:
It seems we don't need the `capturedElement` here ?

```
if (activeViewTransition->namedElements().find(checkingContext.pseudoElementNameArgument))
     return  activeViewTransition->namedElements().size() == 1
```

Reply:
Sure but then the 2 liner becomes a 3 liner:
```
if (RefPtr activeViewTransition = element.document().activeViewTransition()) {
     if (activeViewTransition->namedElements().find(checkingContext.pseudoElementNameArgument))
          return  activeViewTransition->namedElements().size() == 1
     return false;
}
```

Reply:
oh I think I see what you mean